base/math.jl

```diff
@@ -776,9 +776,10 @@ end
 ldexp(x::Float16, q::Integer) = Float16(ldexp(Float32(x), q))
 
 """
-    exponent(x) -> Int
+    exponent(x::BigFloat)
+    exponent(x::T) where T<:Union{Float16, Float32, Float64}
 
-Get the exponent of a normalized floating-point number.
+Returns the x for 2^y which is closest to x rounded toward zero
```

Comment:
```suggestion
Returns the `y` for `2^y` which is closest to `x` rounded toward zero.
```
? Also, I don't think removing the previous explanation is a good idea. It is just another way of saying it, and may be closer to some users depending on their background.

Reply:
I did the changes